File: templates/tests-go/BREACH-002.go
package compliance

import (
	"testing"
)

func TestEnforceTLSVersion(t *testing.T) {
	config := GetConfig()
	tlsVersion := config.Authentication.TLSVersion
	if tlsVersion != "1.2" && tlsVersion != "1.3" {
		t.Errorf("TLS version must be 1.2 or 1.3, got %s", tlsVersion)
	}
}

File: templates/tests-java/BREACH-002.java
package compliance;

import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;

class TLSVersionTest {
    @Test
    void testEnforceTLSVersion() {
        Config config = Config.load();
        String tls = config.getAuthentication().getTlsVersion();
        assertTrue(tls.equals("1.2") || tls.equals("1.3"),
            "TLS version must be 1.2 or 1.3, got " + tls);
    }
}
